features/draft_environment.feature

```diff
@@ -11,14 +11,14 @@ Feature: Draft environment
     Then I should be on the case study page
     And the page should contain the draft watermark
 
-  @draft
+  @draft @high
```

Comment:
Not sure this is a page we want triggered by Nagios. We had issues with named pages in the Whitehall smoke tests (they'd get unpublished etc.).

Is there a way we could dynamically fetch a page?